lib/engine/minor.rb

```diff
@@ -3,10 +3,10 @@
 require_relative 'assignable'
 require_relative 'abilities'
 require_relative 'entity'
-require_relative 'ownable'
 require_relative 'passer'
 require_relative 'spender'
 require_relative 'operator'
+require_relative 'ownable'
```

Comment:
why is this moved?

Reply:
I think I moved it to have it after operator. Have now moved them so they are alphabetical.